mkdocs.yml

```diff
@@ -184,6 +184,7 @@ nav:
           - Issues: community/contribute/issue.md
           - Discussions: community/contribute/discussion.md
           - Pull Requests: community/contribute/pr.md
+          - Compliance Checks: community/contribute/compliance.md
```

Comment:
The "Contribution" section is more generic, like how to open PRs. This specific documentation should be located under "Compliance" or something similar.
https://aquasecurity.github.io/trivy/v0.49/docs/compliance/compliance/